src/components/PeoplePage.tsx

```diff
@@ -1,31 +1,107 @@
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { getPeople } from '../api';
+
+import { Person } from '../types/Person';
 
 export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [isLoad, setIsLoad] = useState<boolean>(false);
```

Comment:
It is better to make `isLoad` `true` by default. Now there's no need to set it `true` in line `23`